flake8 failure on latest master nixpkgs

```
=================================== FAILURES ===================================
_________________________________ FLAKE8-check _________________________________
/build/vulnix-1.7.1/src/vulnix/output.py:128:9: E117 over-indented
```

--- a/src/vulnix/output.py
+++ b/src/vulnix/output.py
@@ -125,12 +125,12 @@ def output_json(items, show_whitelisted=False):
 
 
 def output(items, json=False, show_whitelisted=False, verbose=False):
-        if json:
-            output_json(items, show_whitelisted)
-        else:
-            output_text(items, show_whitelisted, verbose)
-        if any(i.report for i in items):
-            return 2
-        if any(i.masked for i in items):
-            return 1
-        return 0
+    if json:
+        output_json(items, show_whitelisted)
+    else:
+        output_text(items, show_whitelisted, verbose)
+    if any(i.report for i in items):
+        return 2
+    if any(i.masked for i in items):
+        return 1
+    return 0

